<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresenterWindowClass</class>
 <widget class="QWidget" name="PresenterWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresenterWindow</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QListWidget" name="listWidgetQuestions"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelId">
            <property name="text">
             <string>Id</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditId"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelText">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditText"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelAnswer">
            <property name="text">
             <string>Answer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAnswer"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddQuestion">
        <property name="text">
         <string>Add question</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
